src/userInterface/panels/CommandPanel/CommandPanel.tsx

```diff
@@ -1,10 +1,24 @@
 import React from 'react';
 import classes from './CommandPanel.module.scss';
 import { CommandItem } from '../../components/CommandItem/CommandItem';
-import { Switch } from "antd";
+import { Switch } from 'antd/es';
```

Comment:
these should be antd/es/switch

Reply:
'antd/es/switch' has no exported member 'Switch'
with 'antd/es/switch' we can't use named imports

Reply:
changing to default imports